drivers/bridge/setup_verify.go

```diff
@@ -27,11 +29,14 @@ func setupVerifyAndReconcile(config *networkConfiguration, i *bridgeInterface) e
 		return (*IPv6AddrNoMatchError)(bridgeIPv6)
 	}
 
-	// By this time we have either configured a new bridge with an IP address
-	// or made sure an existing bridge's IP matches the configuration
-	// Now is the time to cache these states in the bridgeInterface.
-	i.bridgeIPv4 = addrv4.IPNet
-	i.bridgeIPv6 = bridgeIPv6
```

Comment:
Any reason we are removing this. I remember there was a reason we have to initialize it here (atleast for the IPv4 case) where setup_ipv4 may not happen if the bridge already had an IP address.